### DISCUSSION TITLE
Use port 8080/tcp

### DISCUSSION THREAD

**Discussion by ooonea** [containrrr/watchtower#687] [Q&A] [answered]
Hi, I have installed watchtower on a Scaleway server.  On the server running Ubuntu 16.04.7 with Docker pre-installed.  I have three containers loaded: thelounge, nginx and letsencrypt-companion.  After installing watchtower , the docker ps command returns this to me: [pox.txt](https://github.com/containrrr/watchtower/files/5586281/pox.txt)
Is it normal for watchtower to use port 8080?  In the examples on the internet there is never any port used.

**Comment by simonaronsson** [comment, marked as answer]
Hi!

You don't have to map that port, but yes - it's normal. It's for the HTTP API mode described [here](https://containrrr.dev/watchtower/arguments/#http_api_mode), which we probably need to document better. 😅

**Reply by ooonea** [reply]
So even if I use the basic command to start watchtower automatically it uses port 8080?  However both netdata and lsof tell me that no process is listening on that port. Thanks a lot ❤️

**Reply by simonaronsson** [reply]
Correct. 👍 This is the reason for what you're seeing, but as you say, unless you enable the HTTP API, nothing will listen on that port anyway. https://github.com/containrrr/watchtower/blob/c917b159da7ff01840bd958c8e5a872021a8b605/dockerfiles/Dockerfile#L17

**Reply by ooonea** [reply]
Perfect, you have been very kind and clear.

**Comment by dreamflasher** [comment]
Is there any way to turn this off?

**Reply by piksel** [reply]
It's not enabled unless you explicitly enable it. The port entry in the Dockerfile has no effect unless you forward it.

**Reply by dreamflasher** [reply]
It seems in the current version it's enabled by default? At least I am running the command from the doc without any env, and in `docker ps` it shows as listening to 8080.

**Reply by piksel** [reply]
No, it's just showing what ports are declared in the Dockerfile, that is not something you can toggle at runtime.

**Reply by quincarter** [reply]
I would like to know how to disable this or change the port. It conflicted with a container I had and affected customers because that port was exposed already for my existing container

**Reply by piksel** [reply]
@quincarter Unless you forward it, the port is not in use. And IF you forward it, you specify what port you use on the host. The port specification in the Dockerfile doesn't cause any conflict.

**Reply by simonaronsson** [reply]
Yeah, so when running it, either omit `-p 8080:8080` (or whatever you might have, or in the case of docker-compose, don't include a port mapping dict.
